scan.py

```diff
@@ -39,7 +40,7 @@ def parse_args():
         const="-",
```

Comment:
Can you change the default to "."(the cwd), since "-"(stdout) doesn't make sense anymore.

Reply:
@jwong101 done!